Makefile.toml

```
@@ -104,21 +104,25 @@ then
     exit 1
 fi
 
+echo "📚 Generating documentation..."
+
 mkdir -p $SCHEMA_FOLDER
 find contracts/*/schema -type f -maxdepth 1 -name '*.json' \
     -exec sh -c 'cp "$@" "$0"' $SCHEMA_FOLDER/ {} +
 
 for SCHEMA in $(ls $SCHEMA_FOLDER); do
     awk "{sub(\"#/definitions\",\"./${SCHEMA}/#/definitions\")} {print}"  ${SCHEMA_FOLDER}/${SCHEMA} > ${SCHEMA_FOLDER}/tmp
+    sed -i ${SED_FLAG} -E '/\`[a-zA-Z]+<[a-zA-Z0-9]+>\`/! s/([a-zA-Z]+<[a-zA-Z0-9]+>)/\`\1\`/g' ${SCHEMA_FOLDER}/tmp
+    
     mv ${SCHEMA_FOLDER}/tmp ${SCHEMA_FOLDER}/${SCHEMA}
 done
 
-echo "📚 Generating documentation..."
 rm -rf $DOCS_FOLDER
 jsonschema2md -d schema -o $DOCS_FOLDER --schema-extension json --schema-out $DOCS_FOLDER/schema
```

To fix that I think we need to add the `sed` command on the generated `md` files : 

```suggestion
jsonschema2md -d schema -o $DOCS_FOLDER --schema-extension json --schema-out $DOCS_FOLDER/schema
sed -i ${SED_FLAG} -E 's/([a-zA-Z]+)<([a-zA-Z0-9]+)>/\<code>\1\&lt;\2\&gt;\<\/code>/g' $DOCS_FOLDER/*.md
```

I've change the regex to add `<code></code>` instead of backtick and add html escaping for `>` and `<`.